src/components/App.js

```diff
@@ -1,20 +1,25 @@
 import React from 'react'
 import {Switch, Route} from 'react-router-dom'
 import './App.css';
-// import Dashboard from './Dashboard'
-// import Header from './Header'
+import Dashboard from './dashboard/Dashboard'
+import Header from './header/Header'
 import Calendar from './calendar/Calendar'
+import Form from './form/Form'
 import NavBar from './navbar/NavBar'
 
 function App() {
   return (
     <div className="App">
-      <NavBar/>
+      <Header/>
       <Switch>
-        <Route path="/" component={App.js}/>
+        <Route exact path="/" component={Dashboard}/>
+        <Route path="/calendar" component={Calendar} />
+        <Route path="/new-event" component={Form} />
+        {/* <Route path="/contacts" component={Contacts} /> */}
+        {/* <Route path="/event/:name" render={blah blah match what not here} /> */}
+        {/* <Route component={NotFound} /> */}
```

Comment:
Thanks for adding these route notes. Definitely will make future route connections easier